README.md

```diff
@@ -46,6 +47,14 @@ The Current LTS Plan is:
     release, the LTS Working Group will select a handful of candidate names
     and submit those for a collaborator vote.
 
+** LTS for the version of OpenSSL used by v0.12 (v1.0.1) is scheduled
+   to expire on 2016-12-31. While the typical maintenance LTS cycle
+   for a node release is 12 months, the LTS for OpenSSL v1.0.1 would
+   expire 3 months before the LTS for v0.12 would expire. The LTS WG
+   has therefore decided that the best approach is to shorten the
+   maintenance period for v0.12 and align it's end date with that of
+   OpenSSL v1.0.1's end date.
+
```

Comment:
It is  better to add an official reference to 
https://www.openssl.org/policies/releasestrat.html
as there is a possibility to change their policy and schedule in the future.